build.gradle

```diff
@@ -3,31 +3,39 @@ plugins {
     id 'java'
     id 'war'
     id 'maven-publish'
-    id 'com.etendoerp.gradleplugin' version '1.1.2'
+    id 'com.etendoerp.gradleplugin' version '1.3.2-SNAPSHOT'
```

Comment:
@sebastianbarrozo I would change the plugin version in the repository once we are sure it works with a release instead of snapshot.